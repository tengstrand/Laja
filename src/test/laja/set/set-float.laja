
#set (pi = 3.14159)
#set (num = -1.23)

#write "target/test-output/set-float.out"  
pi = {pi}...
num = {num}
#end
