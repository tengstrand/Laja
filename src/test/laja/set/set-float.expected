pi = 3.14159...
num = -1.23
